cli/src/main/java/org/adamalang/cli/runtime/Output.java

```diff
@@ -0,0 +1,137 @@
+package org.adamalang.cli.runtime;
+
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.adamalang.cli.Util;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class Output {
+
+    private boolean color = true;
+    private boolean json = false;
+    private ArrayList<ObjectNode> outputArray = new ArrayList<>();
+    public Output(Argument args) {
+        HashMap<String, ArgumentItem> arguments = args.arguments;
+        ArgumentItem color = arguments.get("--no-color");
+        ArgumentItem json = arguments.get("--json");
+        if (json != null)
+            this.json = true;
+        if (color != null)
+            this.color = false;
+    }
+
+    public void add(ObjectNode json) {
+        outputArray.add(json);
+    }
+    public void out() {
+        // In the case of json only
+        if (json) {
+            System.out.println("[");
+            for (int i = 0; i < outputArray.size(); i++) {
+                ObjectNode json = outputArray.get(i);
+                String[] lines = json.toPrettyString().split("\n");
+                for (int j = 0; j < lines.length ; j++) {
+                    String outputStr = "  " + lines[j];
+                    if (i < outputArray.size() - 1 && j == lines.length - 1) {
+                        outputStr += ",";
+                    }
+                    System.out.println(outputStr);
+                }
+            }
+            System.out.println("]");
+            return;
+        }
+        Util.ANSI headingColor = Util.ANSI.Reset;
+        Util.ANSI valueColor = Util.ANSI.Reset;
```

Comment:
Instead of a Reset, it would be better to have a Normal which did nothing during wrap.